src/main/java/seedu/dietbook/DietBook.java

```diff
@@ -114,6 +114,8 @@ public static void main(String[] args) {
                 c.execute(dietBook.manager, dietBook.ui);
             } catch (DietException e) {
                 dietBook.ui.printErrorMessage(e.getMessage());
+            } catch (Exception e) {
+                throw new DietException("Wrong command format!");
```

Comment:
Perhaps consider changing to a more generic message.

Reply:
I feel like this is super generic already because all the input are all "commands" so any typos should be safe to consider as a wrong format of command. "Something went wrong" is too vague of a message i feel? wdyt

Reply:
I see. I was thinking that since all the exceptions are being caught here it could also possibly be catching an error from elsewhere in the system other than the command input. However, if you think that only the command input is likely to result in an exception that is caught here I think you can leave it as such.

Reply:
Hmm runtime error and such rite,,,, orite i will just change it to Something went wrong cos i think we will be catching most of the exceptions for commands.

Reply:
Perhaps you can consider catching other possible generic exceptions that may be thrown like NumberFormatException, StringIndexOutOfBound, IOException before catching Exception so that more specific error messages can be output.